JS/inject.js

```diff
@@ -207,6 +263,25 @@ document.querySelectorAll(".l4V7wb")[0].addEventListener("mouseup", function ()
             buttonClicked()
         });
 
+
+        document.querySelector(".XMjwIe").addEventListener("click",function(){
+            setTimeout(function(){     
+               $(".JAPqpe").append(`
+                <div class="youtubeDiv" class="youtubeBtn">
+                <img src="https://i.ibb.co/0QBkmSh/youtube.png" width=24 style="display: inline-block; margin-right: 10px; margin-bottom: -7px; margin-left:15px;"> 
+                <p style="display: inline-block;">Youtube</p>
+              </div>`)
+
```

Comment:
After injecting this code you will want to try to get the CSS to reprocess the height. Try resizing your browser and you will see that the popup does not have a scroll bar.

I have tried:

document.body.style.zoom = 1.0000001;
setTimeout(function(){document.body.style.zoom = 1;},50); //allow some time to flush the css buffer.

but I think I may be inserting it in the wrong place.